### PR TITLE
Pass active node timeout variable to tests

### PR DESCRIPTION
**Pull request by adamfarley**
This allows us to set the active node timeout once in the build
job, and for it to pass through to any test jobs launched by it.

Blocked on:
- PR 2130 in this repo - done
- PR 2012 in the openjdk-tests repo - done

Lacking any of those will prevent this change from working.

Signed-off-by: Adam Farley <adfarley@redhat.com>

### CODE REVIEW

File: pipelines/build/common/openjdk_build_pipeline.groovy [hunk 1]
```diff
@@ -188,7 +188,8 @@ class Build {
 												context.string(name: 'UPSTREAM_JOB_NAME', value: "${env.JOB_NAME}"),
 												context.string(name: 'RELEASE_TAG', value: "${buildConfig.SCM_REF}"),
 												context.string(name: 'JDK_REPO', value: jdkRepo),
-												context.string(name: 'JDK_BRANCH', value: jdkBranch)]
+												context.string(name: 'JDK_BRANCH', value: jdkBranch)
```
Comment (M-Davies) [inline pipelines/build/common/openjdk_build_pipeline.groovy:191]: Missing comma at EOL

**Review by M-Davies — CHANGES_REQUESTED** [review]
One syntax issue, otherwise it looks good

File: pipelines/build/common/openjdk_build_pipeline.groovy (hunk 1 above)
Comment (adamfarley) [reply pipelines/build/common/openjdk_build_pipeline.groovy]: Whoops. Thanks for catching that. Fixed.

**Review by M-Davies — APPROVED** [review]

**Review by karianna — APPROVED** [review]
LGTM

**Comment by adamfarley** [comment]
Thx for the reviews. Will merge after the checks are confirmed green.

**Review by andrew-m-leonard — APPROVED** [review]

**Pull request merged**